pulsar-functions/worker/src/main/java/org/apache/pulsar/functions/worker/rest/WorkerServer.java

```diff
@@ -49,7 +55,17 @@ public WorkerServer(WorkerService workerService) {
     @Override
     public void run() {
         final Server server = new Server(this.workerConfig.getWorkerPort());
-        server.setHandler(WorkerService.newServletContextHandler("/", workerService));
+
+        List<Handler> handlers = new ArrayList<>(2);
+        handlers.add(WorkerService.newServletContextHandler("/admin", workerService));
+        handlers.add(WorkerService.newServletContextHandler("/admin/v2", workerService));
```

Comment:
We shouldn't need to have 2 aliases for same API. Either have `/admin/v2/functions` or `/admin/functions`. I suggest the 1st one for consistency with `/admin/v2/namespaces/`..

Reply:
@merlimat : 

the reason I did this because current function tools extends pulsar admin, pulsar admin still uses v1 api not v2 api. so if I only added in `/admin/v2/functions`, the functions cli will be broken. so I temporarily add it to both v1 and v2 endpoints. we should remove the v1 endpoint after switching pulsar-admin to use v2 endpoints (#1278).

does that make sense?

Reply:
pulsar-admin client cannot completely switch to just use v2 endpoint. Old topics/namespaces will still be using the old endpoint. The Java client will need to be able to use or the other accordingly.

Reply:
okay, so we need to keep this at both `/admin` and `/admin/v2`, no? that would simplify the pulsar-admin client, and when we cut over from v1 to v2, we don't need to change any server side, no?

Reply:
there's no possible cutover from v1 to v2, old topics will stay in same format and we cannot remove in the forseable future.

new topics will use v2 api. functiions, being a new feature will always use v2 api. 

It's fine for now, I will fix when I'll do the v1/v2 choice in client.